src/main/groovy/nextflow/cli/CmdClean.groovy

```diff
@@ -236,7 +243,7 @@ class CmdClean extends CmdBase implements CacheBase {
                     result = false
                     if(!quiet) System.err.println "Failed to remove $dir"
                 }
-                
+
                 result ? FileVisitResult.CONTINUE : FileVisitResult.TERMINATE
```

Comment:
try to avoid to commit empty/unnecessary changes (I know a bit paranoid .. 😅)

Reply:
nah, sorry, you're of course right. I appreciate your comments :)